mappings/net/minecraft/network/packet/s2c/payload/StructuresDebugPayload.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_djjgdtqh net/minecraft/network/packet/s2c/payload/StructuresDebugPayload
+	FIELD f_kdybrwcv ID Lnet/minecraft/unmapped/C_ncpywfca;
+	METHOD m_akcfvyrp writeBlockBox (Lnet/minecraft/unmapped/C_idfydwco;Lnet/minecraft/unmapped/C_hmqypqzz;)V
+		ARG 1 box
+	METHOD m_ohxxgyup readBlockBox (Lnet/minecraft/unmapped/C_idfydwco;)Lnet/minecraft/unmapped/C_hmqypqzz;
+	METHOD m_yfypquja (Lnet/minecraft/unmapped/C_idfydwco;Lnet/minecraft/unmapped/C_idfydwco;Lnet/minecraft/unmapped/C_djjgdtqh$C_obwdgvyc;)V
+		ARG 1 byteBuf
+		ARG 2 piece
+	CLASS C_obwdgvyc PieceQWE
```

Comment:
```suggestion
	CLASS C_obwdgvyc PieceQwe
```

Reply:
I need to release my fix hang on five seconds. once again working around the enigma name conflict bug with that one